codalab/lib/codalab_manager.py

```diff
@@ -341,6 +341,12 @@ def default_user_info(self):
         info['disk_quota'] = formatting.parse_size(info['disk_quota'])
         return info
 
+    def launch_new_worker_system(self):
```

Comment:
rename this from 'new_worker' to just 'worker' and document what this flag means